src/DDoSX/HstsClient.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\DDoSX\Entities\Hsts;
+use UKFast\SDK\DDoSX\Entities\HstsRule;
+
+class HstsClient extends Client
+{
+    const HSTS_MAP = [];
```

Comment:
sanity check - don't need basepath here as we're extending the SDK DDoSX Client 👍

Reply:
I've updated it so that it is matching the other clients and extending the base client with the path